ops/terraform/modules/stateful/main.tf

```diff
@@ -424,23 +424,24 @@ module "replica3_alarms" {
   ok_notification_arn = aws_sns_topic.cloudwatch_ok.arn
 }
 
-# S3 Admin bucket for logs and other adminstrative 
+# S3 Admin bucket for adminstrative stuff
 #
 module "admin" { 
   source              = "../resources/s3"
   role                = "admin"
   env_config          = local.env_config
   kms_key_id          = data.aws_kms_key.master_key.arn
-  acl                 = "log-delivery-write"
+  log_bucket          = module.logs.id
 }
 
-# S3 Admin bucket for logs and other adminstrative 
+# S3 bucket for logs 
 #
-module "elb" { 
+module "logs" { 
   source              = "../resources/s3"
-  role                = "elb"
+  role                = "logs"
   env_config          = local.env_config
-  kms_key_id          = data.aws_kms_key.master_key.arn
+  acl                 = "log-delivery-write"  # For AWS bucket logs
```

Comment:
I've never really seen any best practices on when it's best to use S3 ACLs vs. policies. You?

Reply:
Hmm. I think policies are preferred for most cases (they are finer grained), except for S3 log delivery where ACLs are needed.